src/analysis/souffle/tests/arcs_manifest_tests_todo/README

```diff
@@ -0,0 +1,3 @@
+These tests are imported from the original Arcs repo. They are here because we do not yet correctly
```

Comment:
Can you also mention #64 in the README? There is so much code duplication in the .arcs test files. I wanted to fix this in Arcs, but never got around to it.

Reply:
Done.